Samples/PowerShell/NTA.EnableDisableFlowSources.ps1

```diff
@@ -0,0 +1,26 @@
+# This sample script demonstrates how to enable/disable Flow Source
+# identified by its Caption.
+# You need to be logged as a user with enabled Allow Node Management Rights.
+
+
+# Connect to SWIS
+$hostname = "swis-machine-hostname"                     # Update to match your configuration
+$username = "admin"                                     # Update to match your configuration
+$password = New-Object System.Security.SecureString     # Update to match your configuration
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+$swis = Connect-Swis -host $hostname -cred $cred
+
+
+# Disable Flow Sources
+$nodeCaption = "My test router"
+$nodeId = Get-SwisData $swis "SELECT NodeID FROM Orion.Nodes WHERE Caption = '$nodeCaption'"
+$flowSourcesIds = Get-SwisData $swis "SELECT NetflowSourceID FROM Orion.Netflow.Source WHERE NodeID = $nodeID"
+$flowSourcesIds = $flowSourcesIds |% {[int]$_}
```

Comment:
I would avoid abbreviation in scripts and move this to Foreach-Object even though it's wordier it's easier to read.

Reply:
Fixed